resources/views/shop/shop.blade.php

```diff
@@ -113,42 +113,62 @@
       </div>
     </div>
     <div class="row mx-auto col-md-9 col-sm-8" id="product-data">
-      
```

Comment:
Why this changes were included in this pr?

Reply:
These changes should have been committed earlier, but i forgot to do that and they are here now by mistake.